pyvirtualcam/camera.py

```diff
@@ -108,5 +111,39 @@ def send(self, frame: np.ndarray) -> None:
         _native.send(frame)
 
 
+class _LinuxCamera(CameraBase):
+    def __init__(self, width: int, height: int, fps: float, delay=10, print_fps=False) -> None:
+        super().__init__(width, height, fps, delay, print_fps)
+        self._pyfakewebcam = None
+        index = 0
+        while index < 100:
+            video_device = f'/dev/video{index}'
+            if os.path.exists(video_device):
+                try:
+                    self._pyfakewebcam = pyfakewebcam.FakeWebcam(video_device, width, height)
```

Comment:
This might be bad if the device isn't actually a v4l2loopback device; so we probably need some hint / parameter to control which device to open.
I'd suggest to use an environment variable for now, so we keep the API cross-platform.

Reply:
I would keep the loop as a fall-back if no env var is given, since it's always nice to have things work with minimal configuration if possible. Let's name it `PYVIRTUALCAM_DEVICE` with the value being `/dev/video0` for example.